### DISCUSSION TITLE
Cannot target ARM64 platform on Wix 3.14 Visual Studio 2019 extension

### DISCUSSION THREAD

**Discussion by ori-l** [wixtoolset/issues#8081] [WiX Development]
Hi!
I have an existing msi installer project using Wix 3.11.
I am trying to add support for ARM64 platform and after looking into previous discussions and issues saw that targeting ARM64 should be available since Wix 3.14.

I am using Visual Studio 2019 version 16.11.34 and Wix 3 VS extension.
When I try to add the ARM64 platform Visual Studio claims that the project does not support the platform.
When I looked at the VS extension sources I could not find ARM64 mentioned at all and was wondering if it is even supported? If so is there a special configuration needed or other prequisites? I already have the SDK and can compile other projects.
Also important to mention that I have a custom Wix extension used for my project if that changes anything.

Found this in the sources of the extension:
https://github.com/wixtoolset/VisualStudioExtension/blob/master/src/votive.shared/WixConfigProvider.cs#L291

I know that Wix 4.0 officially supports ARM64 but has major changes in the infrastructure that I want to avoid it if possible

**Comment by bevanweiss** [comment]
I wouldn't expect WiX v3 to have had the significant re-work required to support ARM64... whilst WiX v4+ have been largely re-built.

I think you should move to WiX v5+ if you want any meaningful support for the ARM64 targets.

The migration from v3 to v4+ is something that the WiX team were/are trying to smooth out, depending on how complex your authoring is, you might find that the magic just works... if you're using Heat and other MSBuild magic to bulk capture source files, that might require some additional handling, but I think you'll find it quite straight forward for most situations.
